superbench/runner/runner.py

```diff
@@ -58,7 +58,7 @@ def __get_enabled_benchmarks(self):
                 return list(self._sb_config.superbench.enable)
         return [k for k, v in self._sb_benchmarks.items() if v.enable]
 
-    def deploy(self):
+    def deploy(self):    # pragma: no cover
```

Comment:
What's the meaning of `# pragma: no cover`?

Reply:
it is used to skip coverage calculation in unit tests for specific lines/functions
these functions will be covered in ansible integration tests